lib/reverse_proxy_plug/http_client.ex

```diff
@@ -10,4 +10,7 @@ defmodule ReverseProxyPlug.HTTPClient do
                | __MODULE__.AsyncResponse.t()
                | __MODULE__.MaybeRedirect.t()}
               | {:error, error()}
+
+  if !Code.ensure_loaded?(HTTPoison) && !Code.ensure_loaded?(Tesla),
```

Comment:
@axelson @Fedjmike WDYT if we remove this check?

In theory anyone can implement this behaviour with whatever HTTP Client they have available. HTTPoison and Tesla were provided just as retrocompatibility and proof-of-concept respectively.

I could open a PR if you see fit.

Perhaps we could introduce a runtime check for the `:reverse_proxy_plug, :http_client` config if it doesn't exist already

Reply:
Yes, we should be extensible here. I'd welcome a PR on this.

Reply:
Ok! I'll send a PR on this later today :)

Reply:
👍 from me too